app/controllers/manager/admins_controller.rb

```diff
@@ -22,11 +24,37 @@ def create
       end
     end
 
+    def edit; end
+
+    def update
+      @admin.assign_attributes admin_params
+      if @admin.changed?
+        @admin.skip_password_validation = true
```

Comment:
Nếu muốn update mà không nhập password thì làm theo wiki của devise
https://github.com/plataformatec/devise/wiki/How-To:-Allow-users-to-edit-their-account-without-providing-a-password

Reply:
em `update` nếu có nhu cầu thay đổi password thì vẫn thay đổi được chứ không phải là bỏ qua hẳn password, bởi vậy trong `view edit` em tạo 1 `button` ẩn 2 `form password` đi chứ không phải bỏ hẳn

Reply:
Nhưng em làm thế này nghĩa là lúc nào cũng bỏ qua validate của password đúng không?
Nếu có trường hợp người dùng nhập password sai validation vẫn update vào à?

Reply:
thực ra nó là 1 biến để e bỏ qua validate lúc không update password thôi ạ, để e sửa lại tên